Documentation/Diagnostics/PH2079.md

```diff
@@ -51,3 +51,5 @@ namespace MyOrg.MyProduct
 Specify the first part of your namespace in the `.editorconfig` file under: dotnet_code_quality.PH2079.namespace_prefix = [OrganizationName].[ProductName] 
 
 The general ways of [suppressing](https://learn.microsoft.com/en-us/dotnet/fundamentals/code-analysis/suppress-warnings) diagnostics apply.
```

Comment:
Please also change the "Enabled by default field in the header

Reply:
PH2079 is actually not a Solution Analyzer, and still enabled by default. I added the link to the discussion just for completeness.